src/AiidaDFTK.jl

```diff
@@ -7,22 +7,22 @@ using InteractiveUtils
 using JLD2
 using JSON3
 using Logging
+using LoggingExtras
 using MPI
 using Pkg
 using PrecompileTools
 using TimerOutputs
 using Unitful
 using UnitfulAtomic
 
-export run_json
```

Comment:
The way you do it now, maybe `run()` should be exported or at least maked as `public` ?

Reply:
Made it public. The name is too generic to export it IMO.